fixtures/martin/quantum-wash.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Quantum Wash",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Brady"],
+    "createDate": "2023-12-19",
+    "lastModifyDate": "2023-12-19"
+  },
+  "links": {
+    "manual": [
+      "https://www.martin.com/en-US/products/mac-quantum-wash"
+    ]
+  },
+  "rdm": {
+    "modelId": 34654457,
```

Comment:
I'm not too familiar with RDM, but this number doens't appear in the manual or on the webpage, and the manual says: "Each MAC Quantum Wash has a factory-set RDM UID (unique identification number)".